pkg/api-manager/handlers.go

```diff
@@ -281,8 +281,9 @@ func (h *Handlers) updateAPI(params endpoint.UpdateAPIParams, principal interfac
 				Message: swag.String("api not found"),
 			})
 	}
-	e.Status = entitystore.StatusUPDATING
+
 	updatedEntity := apiModelOntoEntity(params.Body)
+	updatedEntity.Status = entitystore.StatusUPDATING
```

Comment:
I wonder if we should use the merge struct lib for updates to make sure we are not forgetting fields to copy over: https://github.com/imdario/mergo

Reply:
I'll play with it for future work and see if it is worth porting back to this and base image.